### ISSUE TITLE
[Feature Request] Try to resolve component versions

### ISSUE THREAD

**Issue by syd711** [syd711/vpin-studio#360]
>jw15851 — Today at 3:52 PM
> Also just for information a few other minor issues I ran into during install and initial setup.  As a brand new user with almost no exposure to the software. 
> I had a fully functional install with maybe 40-50 tables.  My B2S files existed in a folder (so they stayed organized and were not mixed into the mess there) inside the tables folder.  And many people (at the suggestion of baller installer which tells you it can go anywhere) may not even have it in the tables folder.
The software would not find it unless I pulled the files out of that folder and dumped them into the top level of the tables folder. 
Being able to tell it where the files are, or, ALSO having it read the info saved when b2s is registered so it knows the location would be cool. (My tables are also in their own subfolders for organizational purposes, but it didn’t seem to mind that)
> The other main issue is that the software wouldn’t see or acknowledge the presence of my slightly older versions of the various software packages (b2s, flexdmd, etc) the only one that recognize the older install was VPX itself.  It was only after it did the update or, if your version perfectly matched the current one available that it would lock on and realize that it’s there. 
> This is another area we’re being able to just manually tell it where the install exists would be nice, and ideally have it be able to identify versions other than the absolute most current one) 
> (I’m guessing here it’s totally dependent on GitHub and if the various authors have that exact version published for which it to match against… Still maybe you could look at the version number on the existing install or something like that to identify it)
> Not complaining, so far this is pretty cool stuff… And I’m hoping it might actually make pup packs work worth a damn on my system, because I can’t figure them out to save my life.  🙂

This is not the first request regarding this. Try again to resolve the existing versions somehow.

**Comment by syd711** [comment]
Not fixed, but better documented.

**Comment by syd711** [comment]
Fixed with 3.0.6

**Comment by syd711** [comment]
Once again: won't fix! The actual versions of the components often do not match with the github release artifacts.
